irohad/wsv_checker/wsv_checker.cpp

```diff
@@ -0,0 +1,1119 @@
+#include <algorithm>
+#include <fstream>
+#include <iostream>
+#include <iterator>
+#include <string>
+#include <string_view>
+#include <utility>
+#include <vector>
+//#include <ranges>
+
+#include "ametsuchi/impl/pool_wrapper.hpp"
+#include "ametsuchi/impl/rocksdb_common.hpp"
+#include "common/result_try.hpp"
+#include "logger/logger.hpp"
+#include "logger/logger_manager.hpp"
+#include "logger/logger_spdlog.hpp"
+#include "main/impl/pg_connection_init.hpp"
+#include "main/impl/rocksdb_connection_init.hpp"
+
+#include <gflags/gflags.h>
+#include <nlohmann/json.hpp>
+
+// using namespace std;
+using std::cout, std::cerr, std::endl;
+using std::ostream;
+using std::string_view;
+using namespace std::string_view_literals;
+// using namespace std::ranges;
+using namespace iroha;
+using namespace iroha::ametsuchi;
+using namespace iroha::ametsuchi::fmtstrings;
+using json = nlohmann::json;
+
+#if 0
```

Comment:
remove plz

Reply:
All the code formatted with the same rules. Why your code shouldn't?

Reply:
removed